cheribsd/qemu/Dockerfile

```diff
@@ -1,8 +1,16 @@
+# Staged build: import, decompress, and convert disk images (and kernel):
```

Comment:
Maybe add a comment somewhere here that we use the staged build to avoid having a layer that contains the decompressed image? Might not be obvious to someone looking at this at some point in the future?